openlibrary/components/BulkSearch/components/BulkSearchControls.vue

```diff
@@ -119,10 +158,96 @@ label input {
     flex: 1;
 }
 
+.sampleBar{
+    float:right;
+    margin:0.25rem;
+}
 textarea {
     width: 100%;
+    height: 120px;
     display: flex;
     resize: vertical;
     box-sizing: border-box;
 }
+.progressCarousel{
+    display:flex;
+    overflow-x:scroll;
+    column-gap:10px;
+}
+.progressCard{
+    background-color:#C7E3FC;
+    padding: 16px;
+    width: min(450px, 66vw);
+    height:fit-content;
+    border-radius:30px;
+    display:flex;
+    column-gap:16px;
+    flex-shrink:0;
+    .info{
+        display:flex;
+        flex-direction:column;
+        height:auto;
+        width:auto;
+        row-gap:10px;
+        .heading{
+            color:#0376B8;
+            margin:0px;
+            h3{
+                margin:0px;
+            }
+            p{
+                margin:0px;
+            }
+        }
+        select{
+            width: 100%;
+        }
+        button{
+            background-color:#0376B8;
+            color:white;
+            border-radius:0.5rem;
+            box-shadow: none;
+            border:none;
+            padding: 0.5rem;
+            transition:  background-color 0.2s;
+            min-width:140px;
+            align-self:center;
+            cursor:pointer;
+        }
+        button:hover{
+            background-color:#014c78;
+        }
+    }
+    .numeral{
+        border-radius: 50%;
+        height:48px;
+        width:48px;
+        background-color:white;
+        color:#0376B8;
+        display:flex;
+        font-weight:bold;
+        justify-content:center;
+        flex-shrink: 0;
+        display:flex;
+        align-items:center;
+        font-size:24px;
+    }
+}
+
+.progressCardDisabled{
+    opacity:50%;
+    .info{
+        button:hover{
+            background-color: #0376B8;
+            color: white;
+            cursor:default;
+        }
+    }
```

Comment:
Using a `[disabled]` selector is generally more common, and lets us avoid repeating the colour.

```suggestion
```